rubicon_ml/repository/base.py

```diff
@@ -709,7 +737,13 @@ def get_dataframes_metadata(
 
         return self._load_metadata_files(dataframe_metadata_root, domain.Dataframe)
 
-    def get_dataframe_data(self, project_name, dataframe_id, experiment_id=None, df_type="pandas"):
+    def get_dataframe_data(
+        self,
+        project_name: str,
+        dataframe_id: str,
+        experiment_id: Optional[str] = None,
+        df_type: Literal["pandas", "dask", "polars"] = "pandas",
```

Comment:
we can probably just infer this and store it in the metadata so users don't have to keep specifying